### ISSUE TITLE
Is the generator out of date?

### ISSUE THREAD

**Issue by alexspeller** [ember-animation/liquid-fire#455]
`ember g transition foo` produces this:

```js
import { animate, stop } from "liquid-fire";

export default function (oldView, insertNewView) {

  // Stop any currently running animation on oldView
  stop(oldView);

  // Fade out the old view
  return animate(oldView, { opacity: 0 })
  // Render the new view
    .then(insertNewView)
  // And fade it in, from opacity 0 to 1
    .then(function(newView){
      return animate(newView, { opacity: [1, 0]});
    });
}
```

This is not like the documentation at all - is the generator outdated?

**Comment by ef4** [comment]
Yes, this is out of date.

**Comment by alexspeller** [comment]
Github is being really weird, I can't seem to make a PR because it can't find the branch. Commit for PR is here though:

https://github.com/alexspeller/liquid-fire/commit/a53dc0d081cd369cc62976f716d3592d3f1a0f67

**Comment by ef4** [comment]
Closed in #456
